### ISSUE TITLE
Fusioncatcher sourceforge files moved

### ISSUE THREAD

**Issue by macroscian** [nf-core/rnafusion#173]
Hi,
Thanks for the pipeline and all the work it entails.  In attempting to download the references via `nextflow run nf-core/rnafusion/download-references.nf` etc, I got an HTTP 404 error "Not Found" when it hits the line

```
wget -N http://sourceforge.net/projects/fusioncatcher/files/data/human_v98.tar.gz.aa
```

Looking at source-forge, it seems only v102 is stored at that URL. 

Thanks - Gavin.

**Comment by matq007** [comment]
Hi Gavin, thanks you for pointing that out. I will ask @ndaniel about this

**Comment by ndaniel** [comment]
I guess, that here FusionCatcher is installed via (bio)conda.
Unfortunately, when a new FusionCatcher version has been released, the conda installation does not work anymore with FusionCatcher. 

It would be great if one would jump and fix the conda installation.

**Comment by matq007** [comment]
So basically, you will always store the latest reference only. In that case I guess we will have to update the pipeline and build the reference for fusioncatcher manually as well. I can try implementing a small trick to always verify if there is a new version on GitHub, in that case we will download the references from sourceforge, otherwise we start the build manually using `fusioncatcher-build` as mentioned in the [docs](https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md#42---manual-installation-manual-installation).

**Comment by ndaniel** [comment]
So basically, you will always store the latest reference only. 

> yes, because of lack of space on my comptuer and on SourceForge.net

In that case I guess we will have to update the pipeline and build the reference for fusioncatcher manually as well. I can try implementing a small trick to always verify if there is a new version on GitHub, in that case we will download the references from sourceforge, otherwise we start the build manually using fusioncatcher-build as mentioned in the docs.

> Probably that would work most of the time but there are `corner` cases when this would break. `fusioncatcher-build` always gets the latest version of Ensembl available at the time when is run (today 22.1.2021 is Ensembl release 102). If one runs `fusioncatcher-build` next year then one will get probably Ensembl release 105 and some parsers from FusionCatcher might work or might not (due to changes in data schema of Ensembl database). Therefore one probably should run `fusioncatcher-build` as soon as a new version of FusionCatcher has been released.

**Comment by matq007** [comment]
>Probably that would work most of the time but there are corner cases when this would break. fusioncatcher-build always gets the latest version of Ensembl available at the time when is run (today 22.1.2021 is Ensembl release 102). If one runs fusioncatcher-build next year then one will get probably Ensembl release 105 and some parsers from FusionCatcher might work or might not (due to changes in data schema of Ensembl database). Therefore one probably should run fusioncatcher-build as soon as a new version of FusionCatcher has been released.

So theoretically, with each release of fusioncatcher we just enforce an emsembl version which is supported by internal parsers. This way we should solve all the problem then. The idea is just to keep as much reproducibility as possible. In your releases you always provide an ensembl version, so I can just keep that in mind whenever we upgrade fusioncatcher in rnafusion. Maybe this could be a solution for this?
